packages/cloud-cognitive/src/components/BreadcrumbWithOverflow/_breadcrumb-with-overflow.scss

```diff
@@ -4,124 +4,134 @@
 // This source code is licensed under the Apache-2.0 license found in the
 // LICENSE file in the root directory of this source tree.
 //
+
+// Standard imports.
+@import '@carbon/import-once/scss/import-once';
 @import '../../global/styles/carbon-settings'; // goes before index as it affects how carbon is used.
 @import '../../global/styles/project-settings';
 
+// Imports of Carbon settings and component styles and other
+// related component styles used by ExampleComponent.
 @import 'carbon-components/scss/components/breadcrumb/breadcrumb';
 @import 'carbon-components/scss/components/overflow-menu/overflow-menu';
 
-$block-class: #{$pkg-prefix}-breadcrumb-with-overflow;
-
-.#{$block-class} {
-  display: block;
-}
+// Define all component styles in a mixin which is then exported using
+// the Carbon import-once mechanism.
+@mixin breadcrumb-with-overflow {
+  $block-class: #{$pkg-prefix}--breadcrumb-with-overflow;
 
-.#{$block-class}--space {
-  position: relative;
-  display: block;
-  width: 100%;
-  white-space: nowrap;
-}
+  .#{$block-class} {
+    display: block;
+  }
 
-.#{$block-class}--space--right {
-  text-align: end;
-}
+  .#{$block-class}--space {
+    position: relative;
+    display: block;
+    width: 100%;
+    white-space: nowrap;
+  }
 
-.#{$block-class}--breadcrumb-container {
-  display: inline-flex; // needed to register a width
-  width: 100%;
-}
+  .#{$block-class}--space--right {
+    text-align: end;
+  }
 
-.#{$block-class}--breadcrumb-container .#{$carbon-prefix}--breadcrumb {
-  flex-wrap: nowrap;
-  width: 100%;
-}
+  .#{$block-class}--breadcrumb-container {
+    display: inline-flex; // needed to register a width
+    width: 100%;
+  }
 
-.#{$block-class}--breadcrumb-container--hidden {
-  // This breadcrumb container is used to measure the width of all displayable breadcrumbs
-  position: absolute;
-  top: -100vh;
-  left: -100vw;
-  max-width: 0; // do not take up space.
-  overflow: hidden;
-  visibility: hidden;
-  pointer-events: none;
-}
+  .#{$block-class}--breadcrumb-container .#{$carbon-prefix}--breadcrumb {
+    flex-wrap: nowrap;
+    width: 100%;
+  }
 
-// TODO: REMOVE borrowed from https://github.com/carbon-design-system/carbon/pull/7085
+  .#{$block-class}--breadcrumb-container--hidden {
+    // This breadcrumb container is used to measure the width of all displayable breadcrumbs
+    position: absolute;
+    top: -100vh;
+    left: -100vw;
+    max-width: 0; // do not take up space.
+    overflow: hidden;
+    visibility: hidden;
+    pointer-events: none;
+  }
 
-// Overflow Menu overrides
-.#{$prefix}--breadcrumb-item .#{$prefix}--overflow-menu {
-  position: relative;
-  width: rem(20px);
-  height: rem(20px);
+  // TODO: REMOVE borrowed from https://github.com/carbon-design-system/carbon/pull/7085
```

Comment:
What do we need to remove, and why?

Reply:
It would be more accurate if the comment read; no longer needed when the PR is in the version of Carbon we are using, which will be in 10.32.0 as it is now merged.